src/Admin/CommentsGridFieldBulkAction/ApproveHandler.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+namespace SilverStripe\Comments\Admin\CommentsGridFieldBulkAction;
+
+use Colymba\BulkManager\BulkAction\Handler;
+use SilverStripe\Core\Convert;
+use SilverStripe\Control\HTTPRequest;
+use SilverStripe\Control\HTTPResponse;
+
+/**
+ * A {@link Handler} for bulk approving comments
+ */
+class ApproveHandler extends Handler
+{
+    private static $allowed_actions = ['index'];
+
+    private static $url_segment = 'approve';
+
+    protected $xhr = true;
+
+    protected $buttonClasses = 'font-icon-tick';
+
+    protected $destructive = false;
+
+    protected $label = 'Approve';
+
+    /**
+     * @param  HTTPRequest $request
+     * @return HTTPResponse
+     */
+    public function index(HTTPRequest $request)
+    {
+        $ids = [];
+
+        foreach ($this->getRecords() as $record) {
+            array_push($ids, $record->ID);
```

Comment:
I don't have a problem explicitly with using the `array_push` method, but I normally use (and assume the SS code base does to) `$ids[] = $record->ID;` for pushing to an array